### PR TITLE
Enable gitpod

### PR DESCRIPTION
**Pull request by jmMeessen**
Add some configuration files to enable use of Gitpod for editing.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: .mvn/maven.config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-Pconsume-incrementals
+-Pmight-produce-incrementals
+-Dchangelist.format=%d.v%s
```
Comment (dduportal) [inline .mvn/maven.config:3]:
```suggestion
-Dchangelist.format=%d.v%s

```
Nitpick :)

File: .mvn/extensions.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
```
Comment (jglick) [inline .mvn/extensions.xml:1]: Meant for #26?

File: .mvn/maven.config (hunk 1 above)
Comment (jglick) [inline .mvn/maven.config:3]: ditto

File: .mvn/gitpod-mvn-settings.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <!-- Give access to Jenkins plugins -->
+    <profile>
+      <id>jenkins</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <repositories>
+        <repository>
+          <id>repo.jenkins-ci.org</id>
+          <url>https://repo.jenkins-ci.org/public/</url>
```
Comment (jglick) [inline .mvn/gitpod-mvn-settings.xml:20]: Rather unfortunate Gitpod requires all this duplication. And can we set such stuff at an org level in https://github.com/jenkinsci/.github?

**Pull request merged**